harness/helpers.ts

```diff
@@ -234,6 +220,11 @@ export class TestClient {
     }
 
     async callOnClientInitialized() {
+        const scope = getServerScope()
```

Comment:
you should just leave this method as is, I deliberately didn't use the callback registration approach here because its already a promise-returning method so we can just send `isAsync: true`